docs/life_of_a_godwoken_transaction.md

```diff
@@ -7,48 +7,51 @@ Godwoken is a layer 2 rollup framework for [Nervos CKB](https://github.com/nervo
 
 # Rollup Layer 2 Chain
 
-Fundamentally, Godwoken is a layer 2 solution for layer 1 Nervos CKB blockchain. [Here](https://explorer.nervos.org/aggron/address/ckt1qjd66s4u2xr3djsxahrwnmvkx94tl8j294klugw3ju2nv6h4krhc2t7ddtlvvtj44q4p7w4jlhfzyal5rlwlr9gakvqlxdkqrydv2h22ms4grg40q7w9656uvwk942srzsnep7f4hf5ncqg9pru2mkwv9ctsjp65tp) shows one deployment of Godwoken on CKB's testnet. A new layer 2 Godwoken block, packed in a layer 1 CKB transaction, is issued to CKB's network roughly every 90 seconds. Note the layer 2 block interval is a tunable parameter, which we will explain below. The current state of the layer 2 Godwoken blockchain is committed in the cell data part of a single CKB cell. [This layer 1 CKB transaction](https://explorer.nervos.org/aggron/transaction/0x51bee60d0e68bf3735dbaf7ffeea44727cd2112058a22b5a3e259810ce01a0de) serves as an example of the layer 2 Godwoken block. The chain state, as of this layer 2 block, is stored in the cell data part of the first output cell in this transaction:
+Fundamentally, Godwoken is a layer 2 solution for layer 1 Nervos CKB blockchain. [Here](https://pudge.explorer.nervos.org/address/ckt1qpuljza4azfdsrwjzdpea6442yfqadqhv7yzfu5zknlmtusm45hpuqgqcfn63wfumts4lvrryhc35u43q3aaf5euqq6zeas9) shows one deployment of Godwoken on CKB's testnet. New layer 2 Godwoken blocks are submitted to layer 1 CKB in **block submission transactions**. The current state of the layer 2 Godwoken blockchain is committed in the cell data part of the CKB cell generated by the block submission transaction. [This transaction](https://pudge.explorer.nervos.org/transaction/0x9fd03b09d27326aa54dc53e11d7f7c01f27e570bdc645233b91664323cac0568) is an example of a block submission transaction. The chain state, as of this layer 2 block, is:
```

Comment:
Godwoken is fundamentally a layer 2 solution for layer 1, the Nervos CKB blockchain. [Here](https://pudge.explorer.nervos.org/address/ckt1qpuljza4azfdsrwjzdpea6442yfqadqhv7yzfu5zknlmtusm45hpuqgqcfn63wfumts4lvrryhc35u43q3aaf5euqq6zeas9) is an example of a Godwoken deployment on the CKB testnet. New layer 2 blocks are submitted to layer 1 in **block submission transactions**. The current state of the Godwoken blockchain is committed in the cell data of the CKB cells generated by the block submission transactions. [This transaction](https://pudge.explorer.nervos.org/transaction/0x9fd03b09d27326aa54dc53e11d7f7c01f27e570bdc645233b91664323cac0568) represents an example of a block submission transaction. The chain state of this layer 2 block is as follows: